unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -66,6 +66,82 @@ export default class ActionBinder {
     }
   }
 
+  async getBlobData(file) {
+    try {
+      const objUrl = URL.createObjectURL(file);
+      const response = await fetch(objUrl);
+      if (!response.ok) {
+        throw new Error(`Failed to fetch blob: ${response.statusText}`);
+      }
+      const blob = await response.blob();
+      URL.revokeObjectURL(objUrl);
+      return blob;
+    } catch (error) {
+      //handle error
+      console.error('Error fetching blob data:', error);
+      return;
+    }
+  }
+  
+
+  async uploadFileToUnity(storageUrl, id, blobData, fileType) {
+    const uploadOptions = {
+      method: 'PUT',
+      headers: { 'Content-Type': fileType },
+      body: blobData,
+    };
+    const response = await fetch(storageUrl, uploadOptions);
+    if (response.status != 200) {
+      //Send an event to DC to handle error
+      return;
+    }
+  }
+
+  async chunkPdf(assetData, blobData, filetype) {
+    const totalChunks = Math.ceil(blobData.size / assetData.blocksize);
+    const uploadPromises = [];
+    if(assetData.uploadUrls.length !== totalChunks) {
+      //handle error incorrect temp url to upload chunk
+      return;
+    }
+    for (let i = 0; i < totalChunks; i++) {
+      const start = i * assetData.blocksize;
+      const end = Math.min(start + assetData.blocksize, blobData.size);
+      const chunk = blobData.slice(start, end);
+
+      const url = assetData.uploadUrls[i];
+      uploadPromises.push(this.uploadFileToUnity(url.href, assetData.id, chunk, filetype));
+}
+    // Wait for all uploads to complete
+    await Promise.all(uploadPromises);
+  }
+
+  async continueInApp(assetId, filename, filesize, filetype) {
+    const cOpts = {
+      assetId: assetId,
+      targetProduct: this.workflowCfg.productName,
+      payload: {
+        languageRegion: this.workflowCfg.langRegion,
+        languageCode: this.workflowCfg.langCode,
+        verb: this.workflowCfg.enabledFeatures[0],
+        assetMetadata: {
+          [assetId]: {
+            name: filename,
+            size: filesize,
+            type: filetype,
+          }
+        },
+      },
+    };
+    const response = await this.serviceHandler.postCallToService(
+      this.acrobatApiConfig.connectorApiEndPoint,
```

Comment:
Is this defined somewhere? Should this be added into a new `this.acrobatApiConfig.acrobatEndpoint.connector` key value pair?

Reply:
As discussed marking this as resolved